lib/idempotence/sequential/fixtures.rb

```diff
@@ -0,0 +1,5 @@
+require "entity_projection/fixtures"
+
+require 'idempotence/sequential'
+require 'idempotence/sequential/fixtures/controls'
```

Comment:
I think controls shouldn't be required here, just in the tests.

Reply:
agree

Reply:
@AlfonsoUceda @arepnikov Me too, but the fixture needs the message control